k8s/values.yaml

```diff
@@ -42,3 +35,10 @@ applications:
       hosts:
         - host: penncoursealert.com
           paths: ["/"]
+
+cronjobs:
+  - name: load-courses
+    schedule: "0 3 * * *"
```

Comment:
Just confirming this is supposed to run at 3 am every day

Reply:
Yep